apps/web/src/views/Notifications/index.tsx

```diff
@@ -26,7 +26,7 @@ const Notifications = () => {
   const toast = useToast()
 
   const isW3iInitialized = useInitWeb3InboxClient({
-    projectId: DEFAULT_PROJECT_ID,
+    projectId: 'e542ff314e26ff34de2d4fba98db70bb',
```

Comment:
can we use env variable for this as well? Remeber to add to .env.example together with comment describing the usage